lib/actions/push.ts

```diff
@@ -152,6 +153,12 @@ export const push: CommandDefinition<
 			$ balena push 10.0.0.1 --service my-service
 			$ balena push 10.0.0.1 --system
 			$ balena push 10.0.0.1 --system --service my-service
+
+			$ balena push 23c73a1.local
+			$ balena push 23c73a1.local --source <source directory>
+			$ balena push 23c73a1.local -s <source directory>
+			$ balena push 23c73a1.local --service my-service
+			$ balena push 23c73a1.local --system --service my-service
```

Comment:
These 14 lines of examples between myApp / 10.0.0.1 / 23c73a1.local are a bit too many, no? I suggest spreading the different examples of flags between the IP address and domain name examples, so having 3 examples for myApp, 3 for IP address and 3 for local domain.

Reply:
Done